package.json

```diff
@@ -94,6 +94,7 @@
     "suiteName": "jest-tests"
   },
   "dependencies": {
+    "lodash.uniqueid": "^4.0.1",
```

Comment:
this dep needs to be in the `package/package.json` since it's a dependency of the components and not of styleguidest.